### ISSUE TITLE
7$: Missing syntax error: x += ;

### ISSUE THREAD

**Issue by strager** [quick-lint/quick-lint-js#179]
The following code has a syntax error, but quick-lint-js doesn't emit an error:
```javascript
let x = 3;
x += ;
```

**Comment by keyehzy** [comment]
I'm trying this one out and should send a PR soon.

**Comment by strager** [comment]
@keyehzy fixed this issue in commit 02fce249.
